client/src/pages/locations/GeoLocation.js

```diff
@@ -88,8 +85,7 @@ function fnRequiredWhenEditable(props, propName, componentName) {
 }
 
 GeoLocation.propTypes = {
-  lat: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  lng: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  coordinates: PropTypes.object,
```

Comment:
The coordinates proptype should be more specific.
Check this example: https://github.com/NCI-Agency/anet/blob/48ac66213c1664992d1616755bcec317d876fe9c/client/src/components/SearchFilters.js#L48
and add a CoordinatesPropType with the proper shape.